types/history/history-tests.ts

```diff
@@ -62,7 +62,9 @@ let input = { value: '' };
 }
 
 {
-    let history: MemoryHistory<{ the: 'state' }> = createMemoryHistory();
+    let history: MemoryHistory<{ the: 'state' }> = createMemoryHistory({
+        initialEntries: ['/', { pathname: 'pathname' }],
```

Comment:
this is wrong, that object is expected to have 'key' according to a snip you've posted, `entry.key`.

Reply:
not just key. The object is the same as the location. Please see this URL. The `path` object is the same as the location.

https://github.com/remix-run/history/blob/1b3ae1b5abda4fbb754c869737d02baaad358194/modules/LocationUtils.js#L14-L33